src/scripts/bootstrap_trigger/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM python:3.9.10
+
+RUN pip install boto3==1.16.33 pandas==1.1.5 synapseclient==2.5.1 pyarrow==4.0.0
+RUN git clone -b etl-129 https://github.com/Sage-Bionetworks/BridgeDownstream.git /root/BridgeDownstream
```

Comment:
will update this branch after it gets merged?

Reply:
I added a commit which removes this branch reference.